pkg/azurecluster/builder.go

```diff
@@ -0,0 +1,68 @@
+package azurecluster
```

Comment:
Since this is tightly coupled with testing, should something related be set as in-between package? For example `pkg/test/azurecluster` ? It is a bit unfortunate that language already allocated `testing` which is mandatory pretty much always, but but maybe the singular form works?

Reply:
what about `internal/test/azurecluster`?

Reply:
Works for me. :+1: